src/models/static_injection_subsystem.jl

```diff
@@ -0,0 +1,33 @@
+"""
+Abstract type for a subsystem that contains multiple instances of StaticInjection
```

Comment:
I think we need to clarify that this is applicable to Subsystems that compose StaticInjection. Otherwise, components that have arrays of devices could also apply to the description